### PR TITLE
feat: basic pre and post match statistics [ch541]

### PR DESCRIPTION
**Pull request by Jarrydcrawford**
I'll probably break this in to a few PRs.

### CODE REVIEW

File: src/graphql/fragments.ts [hunk 1]
```diff
@@ -45,6 +45,14 @@ export const TEAM_FRAGMENT = gql`
         }
       }
       votingStatus
+      goalsFor
+      goalsAgainst
+      scorers {
+        player {
```
Comment (harrygr) [inline src/graphql/fragments.ts]: Can we avoid querying too many nested fields on the team fragment. The Team query is used to list many matches so it gets pretty expensive if we have loads of nested queries. I'm thinking we should have seperate queries for a match that is only run when a single match is viewed.

Comment (Jarrydcrawford) [reply src/graphql/fragments.ts]: Yeah definitely. I've done this purely to speed me up during dev.

File: src/utils/ordinals.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const numberToOrdinal = (n: number) =>
+  `${n}${[, 'st', 'nd', 'rd'][(n % 100 >> 3) ^ 1 && n % 10] || 'th'}`;
```
Comment (Jarrydcrawford) [inline src/utils/ordinals.ts]: Shamelessly ripped from SO.

File: src/pages/Match/utils.ts [hunk 3]
```diff
@@ -1,9 +1,75 @@
...
+  if (oldScoreIndex > newScoreIndex) {
+    return 1;
+  }
+
+  if (oldScoreIndex < newScoreIndex) {
+    return -1;
+  }
+
+  return 0;
+};
```
Comment (Jarrydcrawford) [inline src/pages/Match/utils.ts]: This util will be used to determine what state to show the closed match scores/mom/dod votes (i.e. up chevron, down chevron, etc)

File: src/pages/Match/components/CurrentRankingsStat.tsx [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+            );
+            const previousScorerEntries = Object.entries(scorerTotals)
+              .sort(sortEntriesAsc)
+              .reverse();
+            const updatedScorerEntries = Object.entries(updatedScorerTotals)
+              .sort(sortEntriesAsc)
+              .reverse();
+            const relevantScorerEntries = !matchIsClosed
+              ? previousScorerEntries
+              : updatedScorerEntries;
```
[the remark this answers is not in the document]
Comment (Jarrydcrawford) [reply src/pages/Match/components/CurrentRankingsStat.tsx]: Yep. Was just trying to get it shipped and scrubbing the complexity tomorrow.

**Pull request closed without merging**